e2e/cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -111,6 +175,28 @@ describe('at-mention', () => {
         cy.getCurrentChannelId().then((id) => {
             offTopicChannelId = id;
         });
+
+        // Set EnableOpenServer to true so users can sign up
+        const newSettings = {
+            TeamSettings: {EnableOpenServer: true},
+        };
+        cy.apiUpdateConfig(newSettings);
+    });
+
+    it('MM7881 Disable default mention notification settings for "case sensitive first name" and "non-case sensitive username"', () => {
+        ignoreUncaughtException();
+
+        // # Signup a new user with an email address and user generated in signupWithEmail
+        signupWithEmailCreateTeam('unique.' + uniqueUserId, 'unique1pw', 'team.' + uniqueTeamId);
+
+        // # enter first name then open up a specific notifications sub sub panel
+        goToNotificationsFillOutName('first_name.' + uniqueUserId);
+
```

Comment:
This step is not required since the above command `loginAsNewUser()` ensures that the user has the first Name pre-populated.